source/includes/documentation/_vue_js_elements.md

```diff
@@ -518,27 +507,47 @@ public class MediaTextCard extends Card {
     }
 ```
 
-![Cards example](../../images/vuetify/cards.png)
+__Card__ component is a versatile component that can be used for anything from a panel to a static image. The card component has numerous helper components to make markup as easy as possible.  
 
-Cards may contain anything, you can inherit the `Card` class and customize it
-the way you want.
+![Cards example](../../images/vuetify/cards.png)
 
+Cards may contain anything, you can inherit the `Card` class and customize it the way you want.
 
 Basically, you have 4 methods: `title`, `subtitle`, `content` and `actions`.
-They return the parts of a card described [here](https://vuetifyjs.com/en/components/cards/#api).
-The `content` method returns a
-card `text` element, but the `text` method is inherited from `UIBaseElement` that why it has a different name.
+They return the parts of a card described [here](https://v2.vuetifyjs.com/en/components/cards/#api).
+The `content` method returns a card `text` element, but the `text` method is inherited from `UIBaseElement` that why it has a different name.
 
```

Comment:
what about "7.	Element code (with reference to framework version - IMPORTANT!)"? :)

Reply:
Fixed